src/main/java/seedu/address/model/company/Industry.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.company;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Tag in the address book. TODO: Javadocs (Shawn)
+ * Guarantees: immutable; name is valid as declared in {@link #isValidIndustryType(String)}
+ */
+public class Industry {
```

Comment:
I don't see the need for this class as it is the same as the Name class, we could just pass in the messageConstraints to signify a different error message if need to.

Reply:
Yes, we would need to pass in the messageConstraints variable to the parent Name class.  Yes I think the code is good enough to merge!